Tests/LinearSolvers/MLMG/GNUmakefile

```diff
@@ -11,12 +11,17 @@ USE_MPI   = TRUE
 #USE_OMP   = TRUE
 USE_OMP   = FALSE
 
+USE_HYPRE = TRUE
```

Comment:
Could we make the default of this be FALSE?

Reply:
Sure. Made the change.